dashdotdb/models/base.py

```diff
@@ -1,4 +1,9 @@
+from typing import TYPE_CHECKING
 from flask_sqlalchemy import SQLAlchemy
 
-
 db = SQLAlchemy()
+
+if TYPE_CHECKING:
+    from flask_sqlalchemy.model import Model
+else:
+    Model = db.Model
```

Comment:
the proper supported is added to sqlalchemy 2.0, we can skip these changes during uv migration:

* remove `types-Flask-Migrate` because it installed `types-flask-sqlalchemy`
* remove plugins = [ "sqlalchemy.ext.mypy.plugin" ] because it's for 2.0
Then `DORADeployment` related changes are not needed